src/components/editMeasure/measureDetails/measureMetadata/MeasureMetadata.test.tsx

```diff
@@ -174,52 +178,52 @@ describe("MeasureRationale component", () => {
     });
   });
 
-  it("should update the rationale input field when a user types a new value", () => {
-    const { getByTestId } = render(
-      <MeasureMetadataForm measureMetadataType="Rationale" />
-    );
+  it("should update the rationale input field when a user types a new value", async () => {
+    render(<MeasureMetadataForm measureMetadataType="Rationale" />);
 
     const input = getByTestId("measureRationaleInput");
     expectInputValue(input, RATIONALE);
 
-    fireEvent.change(input, {
-      target: { value: NEWVALUE },
+    await act(async () => {
+      fireEvent.change(input, {
+        target: { value: NEWVALUE },
+      });
+      await waitFor(() => expectInputValue(input, NEWVALUE));
     });
-
-    expectInputValue(input, NEWVALUE);
   });
 
-  it("should update the author input field when a user types a new value", () => {
-    const { getByTestId } = render(
-      <MeasureMetadataForm measureMetadataType="Author" />
-    );
+  it("should update the author input field when a user types a new value", async () => {
+    render(<MeasureMetadataForm measureMetadataType="Author" />);
 
     const input = getByTestId("measureAuthorInput");
     expectInputValue(input, AUTHOR);
-
-    fireEvent.change(input, {
-      target: { value: NEWVALUE },
+    await act(async () => {
+      fireEvent.change(input, {
+        target: { value: NEWVALUE },
+      });
+      await waitFor(() => expectInputValue(input, NEWVALUE));
     });
-
-    expectInputValue(input, NEWVALUE);
   });
 
   it("should save the rationale information when the form is submitted", async () => {
-    const { findByTestId, getByTestId } = render(
-      <MeasureMetadataForm measureMetadataType="Rationale" />
-    );
+    render(<MeasureMetadataForm measureMetadataType="Rationale" />);
     const input = getByTestId("measureRationaleInput");
-    fireEvent.change(input, {
-      target: { value: NEWVALUE },
+    act(() => {
+      fireEvent.change(input, {
+        target: { value: NEWVALUE },
+      });
     });
     const save = getByTestId("measureRationaleSave");
-    fireEvent.click(save);
-
-    const success = await findByTestId("measureRationaleSuccess");
-    expect(success).toBeInTheDocument();
-    expect(success.textContent).toBe(
-      "Measure Rationale Information Saved Successfully"
-    );
+    act(() => {
+      fireEvent.click(save);
+    });
+    await act(async () => {
```

Comment:
I was under the impression that an update to state is performed on click since we're waiting for something to show on screen, but removing this still works as intended.